doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -40,18 +57,24 @@ The following steps are made more difficult if code is changing in the "develop"
 Conduct Performance Testing
 ---------------------------
 
+|dedicated|
+
 See :doc:`/qa/performance-tests` for details.
 
 Conduct Smoke Testing
 ---------------------
 
+|dedicated|
+
 See :doc:`/qa/testing-approach` for details.
```

Comment:
```suggestion
See :doc:`/qa/testing-approach` for details. 
Refer to the provided regression checklist for the list of items to verify during the testing process.: [https://docs.google.com/document/d/1OsGJV0sMLDSmfkU9-ee8h_ozbQcUDJ1EOwNPm4dC63Q/edit?usp=sharing](Regression Checklist)
```

Reply:
will need to confirm with rest of the team if they agree with this process before committing this suggestion

Reply:
I guess the alternative is to have the regression checklist in the QA Guide but I'm ok with it being in a google doc for now.